app/helpers/schools/dashboard_helper.rb

```diff
@@ -14,6 +10,14 @@ def ect_with_no_mentor_count(school_cohorts)
       school_cohorts.sum { |sc| sc.current_induction_records.ects.where(mentor_profile: nil).count }
     end
 
+    def link_to_participant(participant_profile, school)
+      govuk_link_to(participant_profile.full_name,
+                    school_participant_path(id: participant_profile.id,
+                                            school_id: school.slug),
+                    no_visited_state: true,
+                    class: "govuk-!-font-weight-regular")
```

Comment:
@ltello we’ve decided to keep this link as bold, to make it a bit clearer that this is the header for the card.

```suggestion
                    no_visited_state: true
```